ui/main/src/app/components/share/datetime-filter/datetime-filter.component.ts

```diff
@@ -75,16 +76,17 @@ export class DatetimeFilterComponent implements ControlValueAccessor, OnInit, On
     public onTouched: () => void = () => {
     }
 
-    // Method call when archive-filter.component.ts set value to 0
+    // Method call when archive-filter.component.ts set value to null
     writeValue(val: any): void {
         if (!this.offset) {
             this.disabled = true;
         }
-        this.resetDateAndTime();
-
         if (val) {
```

Comment:
To fit consistency, maybe this could be `if(!!val)`.

Reply:
done